.circleci/config.yml

```diff
@@ -42,7 +42,8 @@ jobs:
       - run:
           name: Compile source without tests
           command: |
-            mvn -DskipITs=true -DskipTests=true clean install
+            mvn -version
+            mvn -DskipITs=true -DskipTests=true clean package
```

Comment:
I think I want to undo the change from `install` -> `package`. Maven should execute the `pacakge` goal on the way to `install`, anyway: deej
[12:07 PM](https://chat.opennms.com/opennms/pl/e3zo98gha7gg9go88x71t4pyjr)






I'd like to get it merged, but it needs (1) reviewed and (2) I think I need to make one more change: I think I should revert the change that changes the mvn goal from install to package.  install should happen after package, but for some reason, this problem only crops up for me when I run package. [Maven lifecycle reference](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference)

This should be left at `install` and I need to try to reproduce this with that goal.

Reply:
I reverted it.